README.md

```diff
@@ -18,6 +18,15 @@ Rust appears to offer us the ability to implement fast, safer code with developm
 
 ## Goals
 
+Our priorities, higher priority items first, are:
+
+| What | Why |
+| --- | --- |
+| Safe, fast, memory-efficient | If it's not fast enough or it hogs memory nobody will ship it. We would accept unsafe - that we can convince ourselves to be safe - if it got us substantial perf. However, if we can have safety AND speed then it's a pretty easy choice to take that path. |
```

Comment:
Do we mean that "safe", "fast", and "memory-efficient" are all equal-priority goals? I wonder if it is worth acknowledging that there are serious tradeoffs to consider *between* these three goals.

Reply:
I had hoped to acknowledge the potential for tradeoffs tradeoffs in the why. Suggestions for how to make this clearer appreciated :)

I'm of the opinion we don't **know** we have to make a major tradeoff here yet. If safe transmute can be blessed or view is fast then we may have no real problem. Is that fair or do we already know enough to say we definitely have to make hard choices?

Reply:
I don't think there are any substantial tradeoffs that _can_ be made here. If unsafe is an option, we can make both methods equally fast. Otherwise, zerocopy is blocked on safe-transmute.

Reply:
> I don't think there are any substantial tradeoffs that _can_ be made here. If unsafe is an option, we can make both methods equally fast. Otherwise, zerocopy is blocked on safe-transmute.

This is true, but given just the 'view' option, there will be some wiggle room on the safe/fast axis.

I've largely abandoned this 'hybrid' idea, but if we *did* want a 'strictly no unsafe'  implementation it might start to be appealing again, since (say) for a given struct we would only bounds-check once for the creation of all fields.

Reply:
Let's aim to be pragmatic. `unsafe` IS an option but we want it to be minimal and simple. For example, I don't have a problem with using transmute when we know all bit patterns are valid, how many bytes we have, and how big our struct is.

If we can depend on a trusted library (typic? zerocopy? others?) and have the unsafe word in that code instead of ours that's fine. If we believe safe-transmute will eventually drop such a library and choose to do it ourselves in the meantime, also fine.

Reply:
It appears "Safe" may have been intrepreted somewhat more strongly than I intended; updated language to hopefully address